routing/endpointregistry.go

```diff
@@ -81,19 +85,59 @@ type RegistryOptions struct {
 	LastSeenTimeout time.Duration
 }
 
+func NormalizeSchemeHost(s, h string) (string, string, error) {
```

Comment:
likely this wants to be in our net/ package

Reply:
OTOH this used to handle eskip endpoints, i.e. its not a generic helper so maybe its better to keep it inside of routing